rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSink.java

```diff
@@ -147,6 +155,9 @@ private Message prepareMessage(IN input) {
         Validate.notNull(value, "the message body is null");
 
         Message msg = new Message(topic, tag, key, value);
+        if (this.msgDelayLevel > RocketMQConfig.MSG_DELAY_LEVEL00) {
```

Comment:
Do you have equals protection in the following logic?

Reply:
> Do you have equals protection in the following logic?
thanks for reply,i can not understand what you mean "equals protection in the following logic"?

Reply:
I think i know what do you mean, using ">=" ?
But i think if delay level equals 0,it's not necessary to set delay level. So i did not use “>=”.

Reply:
correct